backend/src/db/migrations/20241119143026_add-project-descripton.ts

```diff
@@ -0,0 +1,19 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  const hasProjectDescription = await knex.schema.hasColumn(TableName.Project, "description");
+
+  await knex.schema.alterTable(TableName.Project, (t) => {
+    if (!hasProjectDescription) t.string("description");
+  });
```

Comment:
nit: since there is only one operation in this alter you can move the if check outside it to avoid calling it completely if already present

same with `down` below